wscript

```diff
@@ -91,31 +91,31 @@ def options(opt):
             help='Enable ALSA driver',
             conf_dest='BUILD_DRIVER_ALSA')
     alsa.check_cfg(
-            package='alsa >= 1.0.18',
-            args='--cflags --libs')
+            package='alsa',
+            args=['--cflags', '--libs', 'alsa >= 1.0.18'])
     firewire = opt.add_auto_option(
             'firewire',
             help='Enable FireWire driver (FFADO)',
             conf_dest='BUILD_DRIVER_FFADO')
     firewire.check_cfg(
-            package='libffado >= 1.999.17',
-            args='--cflags --libs')
+            package='libffado',
+            args=['--cflags', '--libs', 'libffado >= 1.999.17'])
     freebob = opt.add_auto_option(
             'freebob',
             help='Enable FreeBob driver')
     freebob.check_cfg(
-            package='libfreebob >= 1.0.0',
-            args='--cflags --libs')
+            package='libfreebob',
```

Comment:
There is no freebob anymore. It was completely replaced by firewire / FFADO

Reply:
I'm not sure what to do.
Do you want me to fix it in this branch or issue a separate pr so it's fixed in master?

Reply:
It was just a hint that the current wscript in the master branch has removed all code related to Freebob.

Reply:
oh gotcha. I've rebased the code.